git-extra/git-extra.install

```diff
@@ -99,30 +99,36 @@ GITATTRIBUTES
 	case "$uname_m" in
 	i686)
 		arch=i686
-		otherarch=x86-64
+		otherarch=x86_64
 		otherpacman=git-for-windows-mingw64
 		;;
 	x86_64)
-		arch=x86-64
+		arch=x86_64
 		otherarch=i686
 		otherpacman=git-for-windows-mingw32
 		;;
 	esac
 
+	grep -q ! wingit.blob.core.windows.net ||
```

Comment:
I'm not sure what this is trying to do

Reply:
Yes, I am still in the process of developing this ;-)

Reply:
@jeremyd2019 _now_ it's ready 😀